components/NavigationBar/examples/NavigationBarLayoutExample.m

```diff
@@ -106,15 +106,43 @@ - (void)viewDidLoad {
   [self.view addSubview:self.trailingItemField];
   [self.view addSubview:self.titleField];
 
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    [self.view.safeAreaLayoutGuide.topAnchor constraintEqualToAnchor:self.navigationBar.topAnchor].active = YES;
+  }
```

Comment:
What happens if we're building for iOS 11 but running on iOS 10? It looks like we're not adding any constraints in that case.

Reply:
thanks! good call.